client/client.go

```diff
@@ -685,7 +685,11 @@ func (client *Client) Debug(action string, params interface{}, result interface{
 	return err
 }
 
-func (client *Client) DebugGet(aspect string, result interface{}) error {
-	_, err := client.doSync("GET", "/v2/debug", url.Values{"aspect": []string{aspect}}, nil, nil, &result)
+func (client *Client) DebugGet(aspect string, params map[string][]string, result interface{}) error {
```

Comment:
I think it'd be clearer from the caller if `params` were a `map[string]string`, and then you did
```go
for k, v := range params {
        urlparams.Set(k, v)
}
```
because then instead of
```go
if err := x.client.DebugGet("change-timings", map[string][]string{"chg-id": []string{chgid}}, &timings); err != nil {
```
you'd have
```go
if err := x.client.DebugGet("change-timings", map[string]string{"chg-id": chgid}, &timings); err != nil {
```

OTOH it's a rather minor thing.

Reply:
On the *other* other hand, you could instead make it
```go
func (client *Client) DebugGet(aspect string, result interface{}, params ...string) error {
```
and the change would be smaller and tidier...

Reply:
Thanks, I like your first suggestion (and followed it), but not completely sold on the last one - it either means I've to assume even number of params (key, value...) or unkeyed values, both seem a bit unclear to the caller; happy to reconsider.